src/stream/src/executor/backfill/snapshot_backfill/executor.rs

```diff
@@ -224,7 +238,22 @@ impl<S: StateStore> SnapshotBackfillExecutor<S> {
                             upstream_buffer.barrier_count(),
                         );
 
+                        backfill_state.finish_epoch(
+                            self.upstream_table.vnodes().iter_vnodes(),
+                            barrier.epoch.prev,
+                        );
+                        let uncommitted = backfill_state.uncommitted_state();
+                        // TODO: apply to progress state table
+                        drop(uncommitted);
```

Comment:
Can you explain why we need to get the reference of the uncommitted state and drop it immediately?